--- html.h	2025-09-07 15:30:40.627036020 -0500
+++ html.h.new	2025-09-04 23:51:17.182773607 -0500
@@ -864,7 +864,7 @@
 
 void HtmlFloatListAdd(HtmlFloatList*, int, int, int, int);
 HtmlFloatList *HtmlFloatListNew();
-void HtmlFloatListDelete();
+void HtmlFloatListDelete(HtmlFloatList *pList);
 int HtmlFloatListPlace(HtmlFloatList*, int, int, int, int);
 int HtmlFloatListClear(HtmlFloatList*, int, int);
 int HtmlFloatListClearTop(HtmlFloatList*, int);
